src/Razor/src/Microsoft.VisualStudio.RazorExtension/SyntaxVisualizer/SyntaxVisualizerControl.xaml

```diff
@@ -4,28 +4,23 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:local="clr-namespace:Microsoft.VisualStudio.RazorExtension.SyntaxVisualizer"
+             xmlns:vsshell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
+             xmlns:vsui="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
              mc:Ignorable="d"
              d:DesignHeight="450" d:DesignWidth="800"
              Loaded="SyntaxVisualizerControl_Loaded"
              GotFocus="SyntaxVisualizerControl_GotFocus"
              LostFocus="SyntaxVisualizerControl_LostFocus"
-             Unloaded="SyntaxVisualizerControl_Unloaded">
+             Unloaded="SyntaxVisualizerControl_Unloaded"
+             Background="{StaticResource {x:Static vsshell:VsBrushes.ToolWindowBackgroundKey}}">
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="../VSThemeDictionary.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
     <DockPanel>
-        <ToolBarTray IsLocked="true" DockPanel.Dock="Top">
-            <ToolBar AutomationProperties.Name="Config" KeyboardNavigation.TabNavigation="Continue">
-                <ToggleButton x:Name="showSourceMappingsButton" Content="Show C# Transitions" Click="ShowSourceMappingsButton_Click" TabIndex="1" />
-                <Button x:Name="showGeneratedCode" Content="Show Generated C#" Click="ShowGeneratedCode_Click" TabIndex="2" />
-                <Button x:Name="showGeneratedHtml" Content="Show Generated Html" Click="ShowGeneratedHtml_Click" TabIndex="3" />
-            </ToolBar>
-        </ToolBarTray>
-        <TreeView x:Name="treeView" AutomationProperties.Name="Nodes" TabIndex="4" KeyUp="treeView_KeyUp">
-            <TreeView.Resources>
-                <!-- Style the inactive selection the same as active -->
-                <SolidColorBrush x:Key="{x:Static SystemColors.InactiveSelectionHighlightBrushKey}"
-                     Color="{DynamicResource {x:Static SystemColors.HighlightColorKey}}" />
-                <SolidColorBrush x:Key="{x:Static SystemColors.InactiveSelectionHighlightTextBrushKey}"
-                     Color="{DynamicResource {x:Static SystemColors.HighlightTextColorKey}}"/>
-            </TreeView.Resources>
-        </TreeView>
+        <TreeView x:Name="treeView" AutomationProperties.Name="Nodes" />
```

Comment:
Can we put back to the inactive colour? Also, need the event handler, for the enter key to work.. that was one of the Ally items.

Also tabbing doesn't get to the treeview anymore, so guessing the TabIndex needs to come back too?

Reply:
Even with the tab index added back, this regresses https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1579500

Maybe we can refer that to the platform though?

Reply:
When does inactive color show?

Tabbing I'll look at tomorrow for sure. Def need to fix

Reply:
When you're focused on the editor, we still select the node in the treeview, but it doesn't have focus, so its light grey on white, which is bad for contrast (and honestly, is also just hard to see).

Reply:
I see, yea it's still hard to see. I'll put this back. It shows up fine in both blue and dark theme with testing